src/main/java/fr/xephi/authme/settings/properties/RestrictionSettings.java

```diff
@@ -193,6 +193,22 @@ public final class RestrictionSettings implements SettingsHolder {
     public static final Property<Set<String>> UNRESTRICTED_INVENTORIES =
         newLowercaseStringSetProperty("settings.unrestrictions.UnrestrictedInventories");
 
+    @Comment({
+        "While in unregistered/logged out state, should players be set to spectator mode and",
+        "forced to spectate within a spawn point invisible armor stand, for 0 movement and head",
+        "pitch + yaw? may be more effective than 'allowMovement' at locking the player in place."
+    })
+    public static final Property<Boolean> SPECTATE_STAND_LOGIN =
+        newProperty("settings.restrictions.spectateStandLogin", true);
+
+    @Comment("Head Yaw position for 'spectateStandLogin'.")
+    public static final Property<Double> HEAD_YAW =
+        newProperty("settings.restrictions.headYaw", 0.0f);
+
+    @Comment("Head Pitch position for 'spectateStandLogin'.")
+    public static final Property<Double> HEAD_PITCH =
+        newProperty("settings.restrictions.headPitch", 0.0f);
```

Comment:
Would it make sense to put these paths under a dedicated path? In the sense of `settings.restrictions.spectateStandLogin.pitch` (or .headPitch). With that naming, settings.restrictions.spectateStandLogin would have to become something like "settings.restrictions.spectateStandLogin.enabled"

Reply:
Yes, it would. It looks much better this way. I put the properties in a separate path